Program.cs

```diff
@@ -1,15 +1,23 @@
-using System.IO;
-﻿using System;
-using System.Net.Sockets;
+using System;
 using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Net;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
 using UdtSharp;
 
 namespace p2pcopy
 {
     class Program
     {
-        static void Main(string[] args)
+        private static int _defaultTaskCount = 10;

```

Comment:
nit: I believe the existing coding style of p2pcopy doesn't use underscores for fields

Reply:
Couldn't find class-level private field in the codebase, however, I'll remove the underscore.